sources/models/src/de.rs

```diff
@@ -207,3 +207,126 @@ mod mirrors_tests {
         );
     }
 }
+
+// This specifies that any non negative i64 integer, -1 and "unlimited"
+/// are the valid resource-limits. The hard-limit="unlimited" or -1 and
+/// soft-limit="unlimited" or -1 sets u64::MAX in spec file for container
+/// that converts to unlimited in containers for that resource
```

Comment:
```suggestion
// This specifies that any non negative i64 integer, -1, and "unlimited"
/// are the valid resource-limits. The hard-limit set to "unlimited" or -1 
/// and soft-limit set to "unlimited" or -1 are converted to u64::MAX in 
/// the spec file for the container runtime which ultimately represents 
/// unlimited for that resource
```

The last sentence was a little hard to understand, does this still mean the same thing or did I mis-understand what you are trying to say?

Reply:
Yeah it means same, will change to the one that you suggested.